### DISCUSSION TITLE
What is the new RPM repository URL ?

### DISCUSSION THREAD

**Discussion by cvgaviao** [nodesource/distributions#1743] [General]
Hello, 
We need to setup a proxy repo on our Artifactory pointing to nodesource RPM one.
I found on my research this URL that is working https://rpm.nodesource.com/pub_20.x/el/9/x86_64/repodata/repomd.xml. If removing `repodata/repomd.xml` from it, I think I can use it on Artifactory.

But I couldn't find any one based on the URL provided on the nodesource main page: 
`sudo yum install https://rpm.nodesource.com/pub_20.x/nodistro/repo/nodesource-release-nodistro-1.noarch.rpm -y
`
Could someone point me to it ? 
thanks

**Comment by cvgaviao** [comment]
@JesusPaz, could you please give me a help here ?

**Comment by zrhoffman** [comment]
Here are the new `repodata/repomd.xml` URLs:
- https://rpm.nodesource.com/pub_20.x/nodistro/nodejs/x86_64/repodata/repomd.xml
- https://rpm.nodesource.com/pub_20.x/nodistro/nodejs/aarch64/repodata/repomd.xml

(or, generally, `https://rpm.nodesource.com/pub_20.x/nodistro/nodejs/$basearch/repodata/repomd.xml`)
